tests/wpt/web-platform-tests/css/cssom-view/scroll-no-layout-box.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>cssom-view - Scrolling element with no layout box</title>
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#dom-element-scroll">
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#css-layout-box">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div style="display: none">
+  <div id="elem"></div>
+</div>
+
+<script>
+setup({explicit_done:true});
```

Comment:
Also, is there any reason this needs `explicit_done`, or waiting for `onload` for that matter?

Reply:
Maybe not, I just cargo-culted the format of the `elementScroll.html` test. I'll have a closer look to see if it's really necessary.

Reply:
I thought it's not necessary. If you look at more test files, you will find some without this. I think it is as same as `onload`, but this case could without it.